src/test/ui/proc-macro/invalid-punct-ident-1.stderr

```diff
@@ -1,3 +1,5 @@
+query stack during panic:
```

Comment:
Please filter this out with more filter instructions in the corresponding `.rs` file, the same with all the other proc macros.

Reply:
sorry I did not understand that what should I do
can you please explain more?

Reply:
Did it not disappear after switching the condition?

Reply:
In case it's still there after the change, what I meant was to add more of the regexes like https://github.com/rust-lang/rust/blob/767e84a834d21eb8acc063f5f89753f4d467f589/src/test/ui/proc-macro/invalid-punct-ident-1.rs#L5-L11 in order to filter out these two new lines.
Basically I think you can add

```
// normalize-stderr-test "query stack during panic:\n" -> "" 
// normalize-stderr-test "end of query stack:\n" -> "" 
```

and these lines should disappear

Reply:
ok got it, thanks, I will fix now